node/utils/metrics/metrics.ts

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios'
 
 const ANALYTICS_URL = 'https://rc.vtex.com/api/analytics/schemaless-events'
```

Comment:
One thing that I saw in the internal-docs, it is the host for ingestion data. We are using rc.vtex but we should use analytics.vtex as [doc](https://internal-docs.vtex.com/Analytics/VTEX-Data-Platform/data-ingestion-API/#schemaless-events-api). After that, I asked for analytics-support and they confirmed the host in the [thread](https://vtex.slack.com/archives/C02T3NYGVNX/p1699469093595349).

Reply:
As discussed offline, apparently "normal" IO apps do not have access to `analytics.vtex.com`, only IO Infra apps.
We decided to keep this URL for now and if we keep seeing errors, we can reachout to #analytics-support and see what can be done.